frontend/src/views/files/Listing.vue

```diff
@@ -367,6 +372,7 @@ export default {
     },
     headerButtons() {
       return {
+        gotoParent: true,
```

Comment:
maybe this could be a preference?
not sure if everyone needs this button, since you haven't added it already.

I'm ok to add it as a preference, and show it only if the preference is enable, if you think that's better.